scrubble-client/src/styles/pages/game.scss

```diff
@@ -0,0 +1,142 @@
+$base-color: #404040;
+$logo-color: white;
+$button-color: lightgray;
+$head-margins: 1rem;
+
+html {
+  box-sizing: border-box;
+}
+*,
+*:before,
+*:after {
+  box-sizing: inherit;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  font-family: Montserrat, sans-serif;
+  margin: 0 auto;
+  height: 100vh;
+  width: 100%;
+}
+
+.header {
+  background-color: $base-color;
+  display: flex;
+  width: 100%;
+  height: 3.2rem;
+  margin: 0;
+
+  &__logo {
+    color: $logo-color;
+    font-weight: 700;
+    font-style: italic;
+    margin-left: $head-margins;
+  }
+}
+
+.room-code {
+  display: inline-flex;
+  margin: 0;
+  align-content: center;
+  &__label {
+    line-height: 0;
+    padding-left: 0.25rem;
+    padding-right: 0.5rem;
+  }
+  &__code {
+    font-weight: 400;
+    line-height: 0;
+    padding-right: 0.25rem;
+  }
+}
+
+.players {
+  width: 100%;
+  max-height: 200px;
+  margin: 1rem 2rem;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+
+  align-content: center;
+
+  &__name {
+    line-height: 0.2;
+    padding-left: 0.25rem;
+    padding-right: 0.5rem;
+  }
+  &__score {
+    font-weight: 400;
+    line-height: 0.2;
+    padding-right: 0.25rem;
+  }
+  &__player {
+    display: inline-flex;
+    border: 1px black solid;
+    margin: 0.25rem;
+  }
+  &__current-turn {
+    display: inline-flex;
+    border: 4px black solid;
+    margin: 0.25rem;
```

Comment:
The reasoning for my previous comment is here, since the only thing that changes between `&__player` and `&__current-turn` is the border (and conceptually they are the same), it'd probably make more sense to arrange these like:
```scss
&__player {
    display: inline-flex;
    border: 1px black solid;
    margin: 0.25rem;
    &--current-turn {
        border: 4px black solid;
    }
}
```

Reply:
yeah this makes way more sense. I hadn't seen this &--  yet, but looks like thats the M of BEM??